doc/content/gateways/lora-basics-station/cups.md

```diff
@@ -18,7 +18,7 @@ Follow instructions for [Adding Gateways in the Console]({{< ref "/getting-start
 
 ## Create an API Key
 
-CUPS requires an API Key with the following rights:
+CUPS does not require an API Key, but it is highly recommended to use one. If you wish to use Token Authentication, create an API Key with the following rights:
```

Comment:
I'm so sorry. The CUPS protocol does not require an API Key but the TTS implementation does. Could you please revert this?